modules/core/include/opencv2/core/bindings_utils.hpp

```diff
@@ -64,6 +64,20 @@ String dumpString(const String& argument)
     return cv::format("String: %s", argument.c_str());
 }
 
+CV_WRAP static inline
+String testOverloadResolution(int value, cv::Point point = cv::Point(42, 24))
```

Comment:
Please use `const reference` for complex types like other OpenCV API does.

Same comment for the overload below.